src/main/java/com/dawn/jat/illuminati/post/entity/PostEntity.java

```diff
@@ -2,18 +2,27 @@
 
 import lombok.Data;
 import lombok.Generated;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
 @Generated
+@RequiredArgsConstructor
 @Data
 @Document(collection = "post")
 public class PostEntity {
 
     @Id
+    @NonNull
     private String id;
-    private String title;
-    private String tag;
-    private String brief;
-    private String time;
+    @NonNull private String title;
+    @NonNull private String brief;
+    @NonNull private String time;
+    @NonNull private String[] tag;
+    @NonNull private String author;
```

Comment:
add slug custom slug for users to find

Reply:
![image](https://user-images.githubusercontent.com/7389488/72679103-8531b700-3ade-11ea-995e-b491a5fad861.png)

Updated